### ISSUE TITLE
Implement airgrab dark mode

### ISSUE THREAD

**Issue by bayological** [mento-protocol/airgrab-interface#73]
**Description**

We must enable the dark mode toggle and update the application to include the dark mode styles

**Comment by RyRy79261** [comment]
@bayological This ticket is completed right?

**Comment by bayological** [comment]
> @bayological This ticket is completed right?

No, in the last grooming sesh we agreed that it does not make sense to implement at this point for a throwaway site so will not be doing it.
